plugins/provisioners/ansible/cap/guest/debian/ansible_install.rb

```diff
@@ -0,0 +1,28 @@
+
+module VagrantPlugins
+  module Ansible
+    module Cap
+      module Guest
+        module Debian
+          module AnsibleInstall
+
+            def self.ansible_install(machine)
```

Comment:
Using PIP (http://docs.ansible.com/intro_installation.html#latest-releases-via-pip) would be a way to not depend on specific PPAs.  That's the solution other ansible-remote type plugins have the done the installation, such as https://github.com/vovimayhem/vagrant-guest_ansible/blob/master/lib/vagrant-guest_ansible/guest_script.sh